packages/frontend/amp/components/elements/Text.tsx

```diff
@@ -31,7 +32,32 @@ export const TextStyle = (pillar: Pillar) => css`
             border-bottom: 1px solid ${pillarPalette[pillar].dark};
         }
     }
-    ${body(3)};
+
+    li {
+        margin-bottom: 6px;
+        padding-left: 20px;
+        p {
+            display: inline;
+        }
+    }
+
+    li:before {
```

Comment:
Just for quick reference. (HAVE I USED THE RIGHT COMMENT 😨)

```suggestion
	/* Bullet point */
    li:before {
```